src/controllers/itemDetails/index.js

```diff
@@ -1115,11 +1149,53 @@ function renderMoreFromSeason(view, item, apiClient) {
     }
 }
 
+function renderMoreFromAlbum(view, item, apiClient) {
+    const section = view.querySelector('.moreFromAlbumSection');
+
+    if (section) {
+        if (item.Type !== 'Audio') {
+            section.classList.add('hide');
+            return;
+        }
+
+        const query = {
+            ParentId: item.AlbumId,
+            Fields: 'ItemCounts,PrimaryImageAspectRatio,BasicSyncInfo,CanDelete,MediaSourceCount',
```

Comment:
AFAIK, this field has been removed.
```suggestion
            Fields: 'ItemCounts,PrimaryImageAspectRatio,CanDelete,MediaSourceCount',
```